scripts/dockerhost/cleanup-dockerhosts

```diff
@@ -14,5 +13,4 @@ do
     echo "Cleaning $hostname"
     sudo docker exec $hostname service docker stop
     sudo docker rm -f $hostname
-    sudo ip link delete $i-ext
```

Comment:
It is getting removed on container rm for me. Anyways moved the link delete to before docker rm